extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCStreamExtractor.java

```diff
@@ -249,13 +250,18 @@ public String getLicence() throws ParsingException {
 
     @Override
     public Locale getLanguageInfo() throws ParsingException {
-        return null;
+        String code = JsonUtils.getString(data, "original_language");
```

Comment:
Is there a reason why you don't just do `data.getString("original_language")`?

Reply:
No, I don't know. Why people created JsonUtils.getString then?

Reply:
I think you could use it to do `JsonUtils.getString(data, "response.crappyRenderer.title.simpleText")` instead of `data.getObject("response").getObject("crappyRenderer").getObject("title").getString("simpleText")`, though I think the issue with it is that it doesn't support getting stuff from a JSON array.